package.json

```diff
@@ -1,6 +1,8 @@
 {
   "private": true,
   "scripts": {
+    "bad-linebreaks": "node ./scripts/bad-linebreaks.mjs",
+    "bad-linebreaks:fix": "node ./scripts/bad-linebreaks.mjs fix",
```

Comment:
this isn't needed, since you can `npm run bad-linebreaks fix`

Reply:
if it was turned into `--fix` it'd match eslint and there'd be no need to self-document it.

Reply:
it's not needed, but strong preference for leaving it.  this is the place where people will look for this when CI fails, and `lint:fix` is a common pattern

Reply:
```suggestion
    "bad-linebreaks:fix": "npm run bad-linebreaks fix",
```
then let's at least do this so we're not repeating the command

Reply:
sure? 🤷